xml/System.Net.Sockets/LingerOption.xml

```diff
@@ -230,9 +230,9 @@
         <Parameter Name="comparand" Type="System.Object" Index="0" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="comparand">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="comparand">The <see cref="T:System.Object"/> to compare with the current <see cref="T:System.Net.Sockets.LingerOption"/>.</param>
```

Comment:
Normally we describe objects, strings and primitive types with natural language instead of a cref:

```suggestion
        <param name="comparand">The object to compare with the current <see cref="T:System.Net.Sockets.LingerOption"/> instance.</param>
```